### ISSUE TITLE
Add ability to link to Hacker News and Reddit comments from within post.

### ISSUE THREAD

**Issue by itsjohncs** [Khan/engblog#12]
Setting up a real comment system may happen at some point, but in the meantime, this is probably even better.

**Comment by itsjohncs** [comment]
Calling out that people can also tweet at @KAEngineering might also be a good idea.

**Comment by evykassirer** [comment]
https://disqus.com/ is pretty easy to set up I think - have you considered it?

**Comment by itsjohncs** [comment]
@evykassirer Yeah, I've thought about disqus. I used to have it on an old incarnation of my [personal site](http://johncs.com) but I took it down because of how much it stuff it injects onto the page (it makes this side bar thing, and kinda makes the site a part of a webring sorta thing.

I'm also not super excited about having an actual comment section on the blog pages themselves, because then we'd want to do some kind of moderation.

**Comment by evykassirer** [comment]
Ah, so you're deliberately not going for comment sections on the page itself. Makes sense!
